pkg/sku/cloud_sku_handler.go

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+package sku
+
+type CloudSKUHandler interface {
+	GetSupportedSKUs() []string
+	GetGPUConfigMap() map[string]GPUConfig
```

Comment:
GetGPUConfigMap -> GetGPUConfig.    Map is a data type name which should not be included in the interface method name.